docs/wallet-integration-guide/src/exchange-integration/overview.rst

```diff
@@ -58,6 +64,7 @@ The following dependency diagrams shows the work items and milestones, which we
 * **Earn app rewards for all CN tokens**: is an additional milestone that improves
   the profitability of your integration by implementing the changes required so that
   your exchange earns app rewards on both withdrawals and deposits of all CN tokens.
+  .. REVIEW: Should that say "...withdrawals of all CN token standard tokens / assets?
```

Comment:
I've opted for now to not do that to avoid having to use it everywhere. It is a bit clunky and it seems that the rest of the WIG also doesn't do so.

Reply:
OK, fair enough